publication/src/main/scala/hmda/publication/reports/util/RaceUtil.scala

```diff
@@ -49,7 +49,7 @@ object RaceUtil {
 
       case Joint =>
         larSource.filter { lar =>
-          (applicantTwoOrMoreMinorities(lar) || coApplicantTwoOrMoreMinorities(lar)) &&
+          (applicantOneOrMoreMinorities(lar) || coApplicantOneOrMoreMinorities(lar)) &&
             (lar.race1 == 5 || coApplicantWhite(lar))
```

Comment:
Shouldn't the `lar.race1 == 5` also make sure the other fields are blank, similar to the `coApplicantWhite` function?

Reply:
Ah, you're probably right. That sentence in the wiki is ambiguous 
```
Applicant Race: 1 or Co-applicant Race: 1 equals White (5) and all else are blank
```
but I think your interpretation is more likely to be the correct one. I'll update this.